migration_steps/validation/api_tests/shared/api_test.py

```diff
@@ -262,6 +262,7 @@ def get_response_object(self, endpoint_final):
                 self.api_log(
                     f"Failed request to {endpoint_final} with status {status_code}."
                 )
+                self.api_log(response_text)
```

Comment:
Don't think we should be putting this out to the logs as that response is a full dump of user data. Even though logs are locked down to us it's usually advised that we're not outputting personally identifiable data to them as anyone with viewer privs can see them.

Reply:
Removed, thanks